sot/symbolic/statement_ir.py

```diff
@@ -135,6 +149,10 @@ def __repr__(self):
 
 @Singleton
 class StatementIRFactory:
+    """
+    It is used to create a singleton StatementIR.
```

Comment:
```suggestion
    It is used to create a StatementIR.
```

StatementIR 不是单例，StatementIRFactory 是单例